code/src/constants/common-messages/en.json

```diff
@@ -33,5 +33,7 @@
   "select.defaultOption": "Select an option...",
   "product.status.visible": "Visible",
   "product.status.hidden": "Hidden",
-  "category.computer": "Computers"
+  "category.computer": "Computer",
+  "category.mobile": "Mobile",
+  "category.tablet": "Tablet"
```

Comment:
Can we reuse translations prefixed with `productsAll`? Maybe we can add plural rules here

Reply:
Used translation from `productsAll`